src/Router.js

```diff
@@ -307,7 +306,7 @@ var Router = React.createClass({
   },
 
   _renderScene(route, navigator) {
-    return cloneElement(route.component, route.props);
+    return createElement(route.component, route.props);
```

Comment:
Hmm may need to think on this one:

cloneElement ~1.7m ops/second
createElement ~0.85m ops/second

May want to figure out a better way than using createElement. Will think on it.